expression/builtin_time.go

```diff
@@ -6542,7 +6542,16 @@ func GetMinSafeTime(sessionCtx sessionctx.Context) time.Time {
 	return getMinSafeTime(sessionCtx, getTimeZone(sessionCtx))
 }
 
+// GetMinSafeTS get minSafeTS
```

Comment:
This comment is weird, please try to avoid it or, add some more meaningful descriptions.

Reply:
Fixed. This modification is not needed now. I reset them to the upstream